tests/fixtures/env.php

```diff
@@ -14,7 +14,7 @@
 require __DIR__.'/security-check.php';
 
 if ('APP_ENV' === $_GET['name'] ?? null): ?>
-    <?=$_ENV['APP_ENV']; ?>
+    <?=$_ENV['APP_ENV'] ?? $_SERVER['APP_ENV']; ?>
```

Comment:
not sure why exactly, but with Symfony CLI Webserver, ENV vars passed to Symfony Process are in $_SERVER not $_ENV, but I think that is right, isn't?

Reply:
Even just `$_SERVER['APP_ENV']` (without the call to `$_ENV` first should do the trick).

Reply:
did not work....